src/SectionsNavigation.Uno.WinUI/FrameSectionsNavigator.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Chinook.StackNavigation;
```

Comment:
It seems like `MultiFrame`, `FrameSectionsNavigator`, and `FrameStackNavigator `are almost identical between the UWP and WinUI implementations. Could we share more code instead of duplicating it? It seems like the only differences are around namespaces and dispatcher. Can't we resolve that using global namespaces and some base classes perhaps?